opennms-doc/guide-admin/src/asciidoc/text/config-files/opennms.properties.adoc

```diff
@@ -0,0 +1,920 @@
+
+[[file-opennms-properties]]
+=== File: opennms.properties
+
+This properties file allows you to set global properties for the opennms application.
+These properties will be set as system properties at the beginning of Java startup.
+Any properties set on the Java command-line with `-Dproperty=value` will be overridden by these values.
+
+[options="autowidth"]
```

Comment:
I'm not sure I like autowidth here because some of the panels are not as wide, making it difficult to scan down the document and see the property names easily. I would prefer width=100%.